lib/cli/writer.js

```diff
@@ -67,6 +68,13 @@ exports.writeImage = (imagePath, drive, options, onProgress) => {
     return fs.openAsync(drive.raw, 'rs+');
   }).then((driveFileDescriptor) => {
     return imageStream.getFromFilePath(imagePath).then((image) => {
+      if (!image.size.final.estimation && image.size.final.value > drive.size) {
```

Comment:
Hmmm, does that mean that the `image` object expected by `isDriveLargeEnough` is different to the `image` object returned by `getFromFilePath` ?
Is that deliberate? (and if so, why?) Could/Should more refactoring be done here?